src/Symfony/Bridge/Doctrine/Form/DoctrineOrmTypeGuesser.php

```diff
@@ -51,28 +52,28 @@ public function guessType($class, $property)
         }
 
         switch ($metadata->getTypeOfField($property)) {
-            case 'array':
+            case Type::TARRAY:
```

Comment:
Not sure if they work as collection form type as well. So lets do that in another PR if necessary.